sympy/polys/domains/domain.py

```diff
@@ -1054,6 +1055,13 @@ def poly_ring(self, *symbols: str | Expr, order: str | MonomialOrder = lex) -> P
         from sympy.polys.domains.polynomialring import PolynomialRing
         return PolynomialRing(self, symbols, order)
 
+    def power_series_ring(self, *symbols: str | Expr, prec: int = 6) -> PowerSeriesRing:
+        """Returns a Power Series ring over ring or field. """
```

Comment:
This should have a better docstring including links to the polys/series section of the docs.

Reply:
Let's make this function private for now (with a leading underscore).

Also the docs for the power series ring domain should note that it is experiemental.

A followup PR can be made for this that tests that all the pieces are working with DomainMatrix and Poly.